src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -1111,15 +1111,15 @@ public void submoduleSync() throws GitException, InterruptedException {
      */
     public SubmoduleUpdateCommand submoduleUpdate() {
         return new SubmoduleUpdateCommand() {
-            boolean recursive                      = false;
-            boolean remoteTracking                 = false;
-            boolean parentCredentials              = false;
-            boolean shallow                        = false;
-            String  ref                            = null;
-            Map<String, String> submodBranch   = new HashMap<>();
-            public Integer timeout;
-            Integer depth = 1;
-            int threads = 1;
+            private boolean recursive                      = false;
+            private boolean remoteTracking                 = false;
+            private boolean parentCredentials              = false;
+            private boolean shallow                        = false;
+            private String  ref                            = null;
+            private Map<String, String> submodBranch   = new HashMap<>();
+            private Integer timeout;
+            private Integer depth = 1;
+            private Integer threads = 1;
```

Comment:
@nre-ableton could you replace this merge commit with a rebase on master and correctly resolve the merge conflict?

Reply:
@darxriggs I have no disagreement with a rebase.  

Sorry that I missed the type change from Integer to int for `threads` in the merge commit.